package.json

```diff
@@ -11,6 +11,9 @@
   "scripts": {
     "start": "gulp & node ./bin/www",
     "dev": "gulp dev",
+    "lint": "eslint --quiet **/**/*.{js,css,scss,pug}",
+    "lint:fix": "eslint --fix .",
+    "prettify": "prettier --write **/**/*.{js,css,scss,pug}",
```

Comment:
I think it would be better if we use a package like [lint-staged](https://www.npmjs.com/package/lint-staged) for adding pre-commit hooks (preventing bad commits before pushing).

Reply:
Yes, we can add that for better code quality.